src/vivarium_gates_nutrition_optimization_child/components/fertility.py

```diff
@@ -53,7 +54,8 @@ def _get_birth_records(builder: Builder) -> pd.DataFrame:
 
         file_path = data_directory / f"scenario_{scenario}_draw_{draw}_seed_{seed}.hdf"
         birth_records = pd.read_hdf(file_path)
-        birth_records["birth_date"] = pd.to_datetime(birth_records["birth_date"])
+        # birth_records["birth_date"] = pd.to_datetime(birth_records["birth_date"])
+        birth_records["birth_date"] = pd.to_datetime("2024-12-31 12:00:01")
```

Comment:
This was to keep the step_backward() consistent; it causes a bug otherwise.

Reply:
If this is a constant it should probably be stored somewhere.

Reply:
it's not necessarily a constant, but it makes more sense to pull the information directly from the configuration I think